packages/info/lib/doctor.js

```diff
@@ -0,0 +1,27 @@
+'use strict';
+
+const define = require('mixinable');
+
+const {
+  sync: { callable: callable, sequence },
+} = define;
+
+const strategies = {
+  diagnoseConfig: callable,
+  diagnoseDuplicatePackages: callable,
+  diagnoseInvalidPackages: callable,
+  submitDiagnosis: callable,
+  getDiagnoses: (...args) => {
+    return sequence(...args).then((results) => [].concat(...results));
+  },
+  logDiagnoses: sequence,
```

Comment:
Should we name this `inspectDiagnoses`? Because logging is only one way of handling it, right?

Reply:
I thought about that, too - but, once again, all current diagnoses only log, and I would want to keep it that way.